tidy3d/plugins/smatrix/component_modelers/modal.py

```diff
@@ -225,7 +226,7 @@ def plot_sim(
             mode_source_0 = self.to_source(port=port_source, mode_index=0)
             plot_sources.append(mode_source_0)
         sim_plot = self.simulation.copy(update={"sources": plot_sources})
-        return sim_plot.plot(x=x, y=y, z=z, ax=ax)
+        return sim_plot.plot(x=x, y=y, z=z, ax=ax, transpose=transpose)
```

Comment:
This function was verified by running the [SMatrix.ipynb](https://github.com/flexcompute/tidy3d-notebooks/blob/develop/SMatrix.ipynb) notebook example.